CMakeLists.txt

```diff
@@ -15,10 +14,13 @@ endif()
 
 if (NOT TG_OWT_SPECIAL_TARGET STREQUAL "")
     set(TG_OWT_PACKAGED_BUILD 0)
+    set(BUILD_SHARED_LIBS OFF)
 else()
     set(TG_OWT_PACKAGED_BUILD 1)
 endif()
 
+option(TG_OWT_USE_PROTOBUF "Use protobuf to generate additional headers. Useful for packaged build." ${BUILD_SHARED_LIBS})
```

Comment:
```suggestion
option(BUILD_SHARED_LIBS "Builds shared libs instead of static" ${TG_OWT_PACKAGED_BUILD})
option(TG_OWT_USE_PROTOBUF "Use protobuf to generate additional headers. Useful for packaged build." ${BUILD_SHARED_LIBS})
```